app/controllers/openid_connect/logout_controller.rb

```diff
@@ -30,26 +30,36 @@ def delete
         analytics.logout_initiated(**result.to_h.except(:redirect_uri))
         irs_attempts_api_tracker.logout_initiated(success: result.success?)
 
+        redirect_user(redirect_uri)
         sign_out
-        if IdentityConfig.store.openid_connect_redirect_interstitial_enabled
-          @oidc_redirect_uri = redirect_uri
-          render(
-            'openid_connect/shared/redirect',
-            layout: false,
-          )
-        else
-          redirect_to(
-            redirect_uri,
-            allow_other_host: true,
-          )
-        end
       else
         render :error
       end
     end
 
     private
 
+    def redirect_user(redirect_uri)
+      if IdentityConfig.store.openid_connect_redirect == :client_side
```

Comment:
ditto re case/when and default branch

Reply:
Thanks, reads a lot better, changed in 5195194